js/bybit.js

```diff
@@ -2420,7 +2573,38 @@ module.exports = class bybit extends Exchange {
 
     parseOrder (order, market = undefined) {
```

Comment:
@ttodua changes inside shared parsers (parseOrder, parseTrade, etc) will definitely clash with @sc0Vu PR. Since you're upgrading spot only, can you please create a dedicated parser for it? in the end spot and derivative structures are different, so in short, would be something like this:

```Javascript
// current implementation stays as is because will be modified in a different PR
parseOrder(order,market) {
}

// new parser that will handle v3 spot upgrade
parseSpotOrder() {

}
```

what do you think?

Reply:
done.